_monitoring-your-cluster/pa/reference.md

```diff
@@ -9,27 +9,26 @@ redirect_from:
 
 # Metrics reference
 
-This page contains all Performance Analyzer metrics. All metrics support the `avg`, `sum`, `min`, and `max` aggregations, although certain metrics measure only one thing, making the choice of aggregation irrelevant.
+OpenSearch provides a number of metrics to help you evaluate performance. The following tables describe the available metrics, grouped by the dimensions that are most relevant for that metric. All metrics support the `avg`, `sum`, `min`, and `max` aggregations, although for certain metrics, the measured value is the same regardless of aggregation type. 
```

Comment:
Wondering if we should mention `Performance Analyzer` here. `OpenSearch Performance Analyzer plugin provides a number of metrics to ...`, The Performance Analyzer plugin can be turned on and off, so it is important to call out that metrics are coming from the plugin